### DISCUSSION TITLE
Any lineage rom for Samsung Galaxy S8+

### DISCUSSION THREAD

**Discussion by trkoehler** [openandroidinstaller-dev/openandroidinstaller#387] [General]
Hi there, I was wondering if someone has a custom rom for the S8+ this would be awesome as I am not an expert on making a rom myself... Any contribution would be highly appreciated... Thanks!!

**Comment by tsterbak** [comment]
The installer does not support the S8+ at the moment, and most of us don't build their own custom ROMs.

That said, I would try this version of LineageOS: https://xdaforums.com/t/lineageos-19-1-android-12l-signature-spoofing-ota-updates-for-s8-s8-note8.4370375/
with the appropriate TWRP for your device 
Snapdragon: https://twrp.me/samsung/samsunggalaxys8plusqcom.html
Exynos: https://twrp.me/samsung/samsunggalaxys8plus.html

You need to add a new config to the installer to use it with the installer: [details here](https://github.com/openandroidinstaller-dev/openandroidinstaller/blob/main/docs/how_to_contribute_your_own_installation_configurations.md)

Good luck and I would be happy about your contribution! :)

**Reply by trkoehler** [reply]
Thanks a lot! Do you by any chance know how to do this on a mac?

**Reply by tsterbak** [reply]
On Mac you can use the executable releases on the release page. But so far we can only build for non-ARM Macs on github. Maybe this will change next year.
For M1/2/3 Macs, the only way is to run the development version from python (basically clone the repo, install poetry and run make install and make app)
